Gruntfile.js

```diff
@@ -16,13 +16,25 @@ var _ = require("lodash");
 var path = require("path");
 var execSync = require("child_process").execSync;
 
+var getFromExec = function (command) {
+    var result;
+    try {
+        result = execSync(command);
+    } catch (e) {
+        console.log("Error executing command: " + command);
```

Comment:
silent: true can also suppress the stderr output by passing "ignore" as the "stdio" option to execSync: 
https://nodejs.org/api/child_process.html#child_process_options_stdio